modules/report-server.nix

```diff
@@ -31,15 +31,18 @@ in {
 
   config = {
 
-    # TODO: remove
-    global.organisation = params.org;
-    global.dnsHostname = mkForce "report-server";
+    global = {
```

Comment:
The TODO should be left in tact, we are mixing nixops options together with a module that describes how to deploy it.

Reply:
I agree, yes.